.github/workflows/tests.yaml

```diff
@@ -62,6 +58,18 @@ jobs:
 
       - run: poetry check --lock
 
+  build:
+    name: Check external build
+    runs-on: ubuntu-latest
+    if: needs.changes.outputs.project == 'true'
+    needs: lockfile
+    steps:
+      - uses: actions/checkout@v4
+
+      - run: pipx run build
+
+      - run: pipx run twine check --strict dist/*
```

Comment:
```suggestion
      - run: pipx run twine check --strict dist/*
      
      - run: pipx install --suffix=@build  --strict dist/poetry-*.tar.gz
      
      - run: poetry@build about | grep $(poetry version --short)
```

Reply:
`pipx install` will build a second time from sdist, which seems wasteful. The version check is simple, but seems like it might be appropriate.